File: app/src/main/java/co/com/ceiba/mobile/pruebadeingreso/storage/cloud/services/UserService.java
package co.com.ceiba.mobile.pruebadeingreso.storage.cloud.services;

import java.util.List;

import co.com.ceiba.mobile.pruebadeingreso.storage.models.User;
import co.com.ceiba.mobile.pruebadeingreso.storage.cloud.Endpoints;
import retrofit2.http.GET;
import rx.Single;

public interface UserService {
    @GET(Endpoints.GET_USERS)
    Single<List<User>> getUsers();
}

File: app/src/main/java/co/com/ceiba/mobile/pruebadeingreso/storage/cloud/services/UserService.kt
package co.com.ceiba.mobile.pruebadeingreso.storage.cloud.services

import retrofit2.http.GET
import co.com.ceiba.mobile.pruebadeingreso.storage.cloud.Endpoints
import co.com.ceiba.mobile.pruebadeingreso.storage.models.User
import retrofit2.Response

interface UserService {
    @GET(Endpoints.GET_USERS)
    suspend fun users(): Response<List<User>>
}